docs/tutorials/chain-deploying.md

```diff
@@ -1,16 +1,16 @@
 ---
 
 layout: single
-title: "Multi Node Chains"
+title: "Tutorials | Multi Node Chains"
```

Comment:
Just standardising the titles of the tutorial documents in line with the use cases and explainers too - either all of them have the section name in, or none of them should do